test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -173,55 +179,38 @@ def test_tfs_model(
 @pytest.mark.model("unknown_model")
 @pytest.mark.neuron_test
 def test_tfs_neuron_model(
-    boto_session,
-    sagemaker_client,
-    sagemaker_runtime_client,
-    model_name,
-    tfs_neuron_model,
-    image_uri,
-    instance_type,
-    accelerator_type,
+    model_name, sagemaker_regions, image_uri, instance_type, accelerator_type
 ):
     input_data = {"instances": [[[[1, 10], [2, 20]]]]}
-    util.create_and_invoke_endpoint(
-        boto_session,
-        sagemaker_client,
-        sagemaker_runtime_client,
-        model_name,
-        tfs_neuron_model,
-        image_uri,
-        instance_type,
-        accelerator_type,
-        input_data,
+    invoke_sm_endpoint_helper_function(
+        ecr_image=image_uri,
+        sagemaker_regions=sagemaker_regions,
+        model_helper=util.find_or_put_model_data,
+        test_function=util.create_and_invoke_endpoint,
+        local_model_path=TFS_NEURON_MODEL_PATH,
+        model_name=model_name,
+        instance_type=instance_type,
+        accelerator_type=accelerator_type,
+        input_data=input_data,
     )
 
 
-@pytest.mark.skip(
-    "CreateEndpointConfig doesn't support trn1: https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ProductionVariant.html#sagemaker-Type-ProductionVariant-InstanceType"
-)
 @pytest.mark.model("unknown_model")
 @pytest.mark.neuronx_test
 def test_tfs_neuronx_model(
-    boto_session,
-    sagemaker_client,
-    sagemaker_runtime_client,
-    model_name,
-    tfs_neuronx_model,
-    image_uri,
-    instance_type,
-    accelerator_type,
+    model_name, tfs_neuronx_model, sagemaker_regions, image_uri, instance_type, accelerator_type
 ):
-    input_data = {"instances": [1.0, 2.0, 5.0]}
-    util.create_and_invoke_endpoint(
-        boto_session,
-        sagemaker_client,
-        sagemaker_runtime_client,
-        model_name,
-        tfs_neuronx_model,
-        image_uri,
-        instance_type,
-        accelerator_type,
-        input_data,
+    input_data = {"instances": [[1.0, 2.0, 5.0]]}
```

Comment:
nit: extra square brackets added, is this intended?

Reply:
Yes, the input needs to be an array of arrays, so that's correct